soak-tests/soak112s/deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -ex
+cd "${0%/*}" # to the folder containing this script.
+
+IMAGE_IDENTIFIER='mesosphere/dcos-ui-soak'
```

Comment:
Shall we add label to distinguish between soak versions? I.e. 'mesosphere/dcos-ui-soak:112s'?

Reply:
Good point, will do